lib/dsn-pcb/dsn-json-to-circuit-json/parse-dsn-to-dsn-json.ts

```diff
@@ -818,14 +826,15 @@ export function processNetwork(nodes: ASTNode[]): Network {
       }
     }
   })
-
   return network as Network
 }
 
 function processNet(nodes: ASTNode[]): Net {
   const net: Partial<Net> = {}
-  if (nodes[1].type === "Atom" && typeof nodes[1].value === "string") {
-    net.name = nodes[1].value
+  // in the smoothie board some of the nets names are numbers like 3.3 so I need to convert them to strings
+  if (nodes[1].type === "Atom") {
+    net.name = nodes[1].value?.toString()
+    debug("net name was not string before parsing", net.name)
```

Comment:
i don't understand this message- you're always executing it so i think it's misleading

Reply:
you should be checking if it's not a string then emitting the `debug("warning: net name was not a string", net.name)`